docker-compose.yml

```diff
@@ -17,13 +17,19 @@ services:
     ports:
       - "11234:5432"
 
+  rabbitmq:
```

Comment:
I figured I'd stick to RabbitMQ since Open Notificaties uses it as well, but I read that Redis can be used as a broker as well, not sure which we would prefer in this case

Reply:
redis - less moving parts because we already have it in the stack and it has less peculiarities that RabbitMQ seems to have. There are also existing recipes for Highly-Available redis clusters (left as an exercise to the devops engineer)